### DISCUSSION TITLE
NFS disaggregation

### DISCUSSION THREAD

**Discussion by mekevans** [mekevans/forestTIME#21] [General]
This is the beginning of a use case discussion on the topic of disaggregating forest C accounting to the NFS unit scale.

**Comment by diazrenata** [comment]
I just added an example (here: https://github.com/mekevans/forestTIME/blob/main/use_cases/nfs/national_forests_az.md) of extracting trees on National Forests in Arizona, using the "OWNCD" from FIADB.

**Comment by mekevans** [comment]
gotcha, so we're currently lacking a way to select a given National Forest

**Comment by mekevans** [comment]
Quick question Renata...I think I could use some kind of read.me or instructions on how to use the interpolation/annualization function. Let's say I wanted to get annual estimates of tree-level above-ground biomass for all trees in a plot, or all trees in the plots in a given national forest (e.g., in Arizona)...where would I start?

**Comment by diazrenata** [comment]
I just updated the readme here: https://github.com/mekevans/forestTIME/tree/main/use_cases/new_packaging with some instructions for a quick-ish start! 

You'll need to clone this repo to your computer, get the right packages installed, and download a copy of the database (link in the readme). Then, for the annualized measurements, you'll want to open the forestTIME R project in RStudio, open `use_cases/new-packaging/annualized_from_db.qmd` and run the code interactively, and make any changes you want to to the filtering or variables. I think you can calculate tree-level biomass from DIA and HT - is that correct, or would you need other variables? 

As the lucky first-ever user, please let me know if any of the steps I've described here aren't 100% clear, or if you run into any sticking points! I try to anticipate possible issues, but it's always possible something will come up!
